### PR TITLE
Ask which device to use provisioning dz2

### PR DESCRIPTION
**Pull request by chrisnojima**
Replaces https://github.com/keybase/client/pull/2042

@keybase/react-hackers

### CODE REVIEW

**Comment by chrisnojima** [comment]
@keybase/react-hackers  bump

File: shared/util/feature-flags.js [hunk 1]
```diff
@@ -5,23 +5,31 @@ import getenv from 'getenv'
...
 }
 
 let features = getenv.array('KEYBASE_FEATURES', 'string', '')
 
 // For compatibility, this is deprecated
-if (getenv.boolish('KEYBASE_TRACKER_V2', false)) { features.push('tracker2') }
-if (getenv.boolish('KEYBASE_ALLOW_LOGIN', false)) { features.push('login') }
+if (getenv.boolish('KEYBASE_TRACKER_V2', false)) { features.push(tracker2Key) }
+if (getenv.boolish('KEYBASE_ALLOW_LOGIN', false)) { features.push(loginKey) }
+if (getenv.boolish('KEYBASE_MOBILE_APPS_EXIST', false)) { features.push(mobileAppsExistKey) }
```
Comment (cjb) [inline shared/util/feature-flags.js]: If these are deprecated, maybe we shouldn't add more.  You can use KEYBASE_FEATURES_MOBILE_APPS_EXIST instead, is that okay?

**Comment by cjb** [comment]
:thumbsup:%C

File: shared/common-adapters/text.desktop.js [hunk 2]
```diff
@@ -79,11 +79,25 @@ export default class Text extends Component {
...
+  small: React.PropTypes.bool,
+  reversed: React.PropTypes.bool,
+  children: React.PropTypes.node,
+  style: React.PropTypes.object,
+  onClick: React.PropTypes.func,
+  inline: React.PropTypes.bool,
+  lineClamp: React.PropTypes.number,
+  className: React.PropTypes.string
+}
+
```
Comment (MarcoPolo) [inline shared/common-adapters/text.desktop.js]: This should die, it's wrong

**Pull request merged**